contractcourt/commit_sweep_resolver.go

```diff
@@ -36,6 +39,9 @@ type commitSweepResolver struct {
 	// chanPoint is the channel point of the original contract.
 	chanPoint wire.OutPoint
 
+	currentReport *ContractReport
```

Comment:
also needs to be initialized in the constructor

Reply:
`initReport` added